src/s2_geo.rs

```diff
@@ -8,6 +9,16 @@ use std::vec::Vec;
 
 const LEVEL_MOD: u8 = 1;
 
+pub trait ECEFExt<N> {
+    fn from(_: Self) -> ECEF<N>;
```

Comment:
I'm curious, wouldn't you normally write `from(self)`?

Reply:
Yes, this is nicer, I adjusted it everywhere.